src/test/java/StringTest.java

```diff
@@ -0,0 +1,32 @@
+import static org.assertj.core.api.AssertionsForClassTypes.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+	@DisplayName("요구사항 1")
+	@Test
+	void split()
+	{
+		String[] result = "1,2".split(",");
+		assertThat(result).containsExactly("1","2");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?